### PR TITLE
jinja2 loaders types

### PR DESCRIPTION
**Pull request by jouve**

### CODE REVIEW

File: third_party/2and3/jinja2/loaders.pyi [hunk 1]
```diff
@@ -15,7 +15,7 @@ class FileSystemLoader(BaseLoader):
     searchpath = ...  # type: Text
     encoding = ...  # type: Any
     followlinks = ...  # type: Any
-    def __init__(self, searchpath: Text, encoding: Text = ..., followlinks: bool = ...) -> None: ...
```
Comment (jouve) [inline third_party/2and3/jinja2/loaders.pyi]: https://github.com/pallets/jinja/blob/master/jinja2/loaders.py#L146

File: third_party/2and3/jinja2/loaders.pyi [hunk 2]
```diff
@@ -38,7 +38,7 @@ class DictLoader(BaseLoader):
 class FunctionLoader(BaseLoader):
     load_func = ...  # type: Any
     def __init__(self, load_func) -> None: ...
-    def get_source(self, environment: Environment, template: Text) -> Tuple[Text, Text, Callable]: ...
+    def get_source(self, environment: Environment, template: Text) -> Tuple[Text,  Optional[Text], Optional[Callable]]: ...
```
Comment (jouve) [inline third_party/2and3/jinja2/loaders.pyi]: https://github.com/pallets/jinja/blob/master/jinja2/loaders.py#L317

File: third_party/2and3/jinja2/loaders.pyi [hunk 3]
```diff
@@ -15,7 +15,7 @@ class FileSystemLoader(BaseLoader):
     searchpath = ...  # type: Text
     encoding = ...  # type: Any
     followlinks = ...  # type: Any
-    def __init__(self, searchpath: Text, encoding: Text = ..., followlinks: bool = ...) -> None: ...
+    def __init__(self, searchpath: Union[Text, Iterable[Text]], encoding: Text = ..., followlinks: bool = ...) -> None: ...
```
Comment (jouve) [inline third_party/2and3/jinja2/loaders.pyi]: https://github.com/pallets/jinja/blob/master/jinja2/loaders.py#L146

**Comment by JelleZijlstra** [comment]
Thanks for the PR!

**Pull request merged**